test/integration/container/multimaster_test.go

```diff
@@ -235,11 +245,11 @@ func TestMultimasterSetup(t *testing.T) {
 	yumRepoIP := sanitizeIP(run(t, "docker", "inspect", "yumrepo", "--format='{{.NetworkSettings.IPAddress}}'"))
 
 	// Start the footloose container "VMs" used for testing:
-	run(t, "footloose", "create", "-c", "../../../examples/footloose/centos7/docker/multimaster.yaml")
-	node0IP := sanitizeIP(run(t, "docker", "inspect", "centos-multimaster-node0", "--format='{{.NetworkSettings.IPAddress}}'"))
-	node1IP := sanitizeIP(run(t, "docker", "inspect", "centos-multimaster-node1", "--format='{{.NetworkSettings.IPAddress}}'"))
-	node2IP := sanitizeIP(run(t, "docker", "inspect", "centos-multimaster-node2", "--format='{{.NetworkSettings.IPAddress}}'"))
-	node3IP := sanitizeIP(run(t, "docker", "inspect", "centos-multimaster-node3", "--format='{{.NetworkSettings.IPAddress}}'"))
+	run(t, "footloose", "create", "-c", "../../../examples/footloose/"+node_os+node_version+"/docker/multimaster.yaml")
+	node0IP := sanitizeIP(run(t, "docker", "inspect", node_os+"-multimaster-node0", "--format='{{.NetworkSettings.IPAddress}}'"))
+	node1IP := sanitizeIP(run(t, "docker", "inspect", node_os+"-multimaster-node0", "--format='{{.NetworkSettings.IPAddress}}'"))
```

Comment:
Why is it okay to use `node0` for all of these now (as opposed to `node1`, `node2`, etc.)?

Reply:
Fixed.